api-client/src/models/DateModel.ts

```diff
@@ -34,32 +35,32 @@ class DateModel {
     // if allDay: only date, else: date + time
     let span = this._allDay
       ? formatter.format(this._startDate, {
-          format: 'LL',
+          format: 'DDD',
         })
       : formatter.format(this._startDate, {
-          format: 'LLL',
+          format: 'DDD ta',
```

Comment:
Hmm. I think this needs to be just `DDD t`

Reply:
@sarahsporck without **a** AM and PM are not appearing